src/user/message.ts

```diff
@@ -387,6 +387,24 @@ export class Message extends Accessory implements Sayable {
     return this.payload.text || ''
   }
 
+  public async recalled (): Promise<Message | null> {
```

Comment:
Please rename `recalled()` to `toRecalled()` because we have already had the following methods:

1. `toContact()`
1. `toFileBox()`
1. `toUrlLink()`

Reply:
Okay, will do.